quickstart.md

```diff
@@ -164,250 +166,51 @@ enclave-cc      enclave-cc      9m55s
 ### Platform Setup
 
 While the operator deploys all the required binaries and artifacts and sets up runtime classes that use them,
-certain platforms may require additional configuration to enable confidential computing. For example, the host
-kernel and firmware might need to be configured.
-See the [guides](./guides) for more information.
```

Comment:
The existing text, "See the guides for more information." reads more clearly to me along with having the link to the directory.